packages/terra-date-picker/tests/nightwatch/date-picker-spec.js

```diff
@@ -68,5 +68,28 @@ module.exports = {
     browser.expect.element('.terra-DatePicker-range > .terra-DatePicker').to.be.present;
   },
 
+  'Creates a hidden input with a name atribute of "test-name-attr" and an empty value attribute when no date is entered': (browser) => {
+    browser.url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/date-picker-tests/default`);
+
+    browser.expect.element('.terra-hidden-date-input').to.have.attribute('name').which.equals('test-name-attr');
```

Comment:
Wow, I gave you bad syntax. I'm sorry. I think this is what I meant.

    browser.expect.element('.terra-hidden-date-input[name="test-name-attr"]')

However, this is just a suggestion. It makes it smaller, but your also spells it all out.